src/main/webapp/script.js

```diff
@@ -47,21 +47,27 @@ function cleanseUrl(url) {
 /**
  * Fetches data and adds to html
  */
-function getChart() {
-  $('form').submit(function() {
+async function getChart() {
+  $('form').submit(async function() {
+    commentStats = await getYouTubeComments();
+    averageScore = commentStats.averageScore;
+    aggregatedValues = commentStats.aggregateValues; 
+
     const CommentSentimentTable = new google.visualization.DataTable();
     CommentSentimentTable.addColumn('string', 'SentimentRange');
     CommentSentimentTable.addColumn('number', 'CommentCount');
```

Comment:
I think we can make this sorting problem a little bit easier if we rely on the GoogleChart doing our sorting.

There are two things that we can do:
1. Add a new hidden column `RangeStart` that will contain the `inclusiveStart` from the Range object. This column can have a `number` type. 
(https://stackoverflow.com/questions/8785894/how-to-hide-column-in-google-charts-table)

2. Use the `sort` method of the DataTable to sort the data according to the value of the `inclusiveStart`.

https://developers.google.com/chart/interactive/docs/reference#DataTable_sort